src/Raven.Studio/wwwroot/App/views/database/status/statistics.html

```diff
@@ -18,7 +18,7 @@ <h2 class="on-base-background">General Database Stats
                 </div>
             </div>
             <div class="col-sm-6 col-lg-4 col-xl-3">
-                <div class="stats-item js-size-tooltip">
+                <div class="stats-item size-tooltip">
```

Comment:
why to removed js- prefix?

Reply:
This was not mandatory as well 
I did that only because I removed the `js-` prefix from the class selector in the _previous comment_